backend/app/Notifications/ResetPasswordNotification.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Notifications;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+class ResetPasswordNotification extends Notification
+{
+    use Queueable;
+
+    /**
+     * The password reset URL.
+     *
+     * @var string
+     */
+    protected $url;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @param  string  $url
+     * @return void
+     */
+    public function __construct(string $url)
```

Comment:
```suggestion
    public function __construct(protected string $token)
```
immediately set class properties from constructor